### PR TITLE
docs: website tweaks

### PR DESCRIPTION
**Pull request by paul-soporan**
**What's the problem this PR addresses?**

- There were some broken links.
- There were some outdated commands on the "package not found" search result page.
- The SEO component didn't include the website image.
- The documentation pages didn't have their own description, which caused them to show up with the website-wide description.
- The package search bar was stickied, which caused the header to take up `38.251912595629776%` of the viewport.

**How did you fix it?**

- I've fixed some broken links.
- I've updated the outdated commands on the "package not found" search result page.
- I've added the website image to the SEO component.
- I've added a description to each documentation page.
- I've unstickied the package searchbar.

I've also experimented with converting the `SEO` component to TypeScript.

**Checklist**

- [x] I have read the [Contributing Guide](https://yarnpkg.com/advanced/contributing).

- [x] I have set the packages that need to be released for my changes to be effective.

- [x] I have verified that all automated PR checks pass.

### CODE REVIEW

**Comment by arcanis** [comment]
> The package search bar was stickied, which caused the header to take up 38.251912595629776% of the viewport.

That's interesting, what does it look like for you ? For me it seemed fairly reasonable:

image

**Comment by paul-soporan** [comment]
image

and

image

**Comment by arcanis** [comment]
I see - yep, that's not ideal. Can we make the pinning conditional based on a media query?

**Comment by paul-soporan** [comment]
Let me try 🤔

**Comment by paul-soporan** [comment]
Done! 👍

**Comment by arcanis** [comment]
Oops 😄

image

File: packages/gatsby/src/pages/index.js [hunk 1]
```diff
@@ -180,10 +173,10 @@ const IndexPage = ({data, searchState, onSearchStateChange}) => {
           <Hero>
             <SectionContent>
               <HeroTitle>
-                Safe, stable, reproducible projects
+        Safe, stable, reproducible projects
```
Comment (arcanis) [inline packages/gatsby/src/pages/index.js:176]: Invalid indent?

Comment (paul-soporan) [reply packages/gatsby/src/pages/index.js:176]: Oops, forgot that the `@typescript-eslint/indent` autofix doesn't reindent text in JSX. 😅 
Fixed!

**Comment by arcanis** [comment]
Note: this PR is still blocked on the broken hero. I suspect something weird is going on with the SSR, since the first render at page load is working properly, but soon after the page renders and the display glitches.

**Comment by paul-soporan** [comment]
I know. I'll try looking into it today! 👍

**Comment by paul-soporan** [comment]
I can't manage to get it to work even with CSS media queries (it seems to ignore them for some reason). 🤔

**Pull request merged**